### DISCUSSION TITLE
q, x, z?

### DISCUSSION THREAD

**Discussion by HironTez** [monkeytypegame/monkeytype#4257] [Questions] [answered]
Hi. I noticed that Monkeytype never shows words with the letters 'q', 'x' and 'z'. Is this how it should work?

**Comment by Miodec** [comment, marked as answer]
You can change the word list in the settinsg (for example English 10k) to get more words, which might include q, x, z.

**Reply by HironTez** [reply]
Thanks for the answer. I already did that. But shouldn't the base list also contain all the letters?

**Reply by tajhans** [reply]
@HironTez The base list is the most common words. Words including q, x, and z aren't very common so they aren't shown. When you change your word list to 10k it includes more, less common, words.
